src/corelibs/U2View/src/ov_mca/McaEditor.h

```diff
@@ -107,10 +113,10 @@ private slots:
     /** Selection state controller. */
     McaEditorSelectionController* selectionController;
 
-    void addEditMenu(QMenu* menu) override;
-    void addAlignmentMenu(QMenu* menu);
-    void addAppearanceMenu(QMenu* menu);
-    void addNavigationMenu(QMenu* menu);
+    void addEditMenu(QMenu *menu) override;
+    void addAlignmentMenu(QMenu *menu);
+    void addAppearanceMenu(QMenu *menu);
```

Comment:
Still tons of non-related reformatting: could you please run a formatter over the changed files?